### ISSUE TITLE
android: app: src: main: res: mipmap App logo is too small

### ISSUE THREAD

**Issue by Rishabh510** [yashlamba/simulate#88]
App logo is too small as viewed in the app drawer. This is caused due to not enough mipmap drawables for the logo. 

Screenshot (Simulate app logo contents are barely visible)
![Screenshot_Lawnchair_20200301-002421](https://user-images.githubusercontent.com/42823059/75613627-ffca0980-5b55-11ea-8cfd-1480cab16705.png)

 - Device: Asus max pro m1
 - OS: Android
 - Screen Width: 440px

**Additional context**
I would like to take up this **issue**.

**Comment by yashlamba** [comment]
Good catch! The logo is really weird with a lot of devices. You can go ahead and work on this!
